graph-server/src/index.js

```diff
@@ -0,0 +1,65 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const expressValidator = require('express-validator');
+const { checkSchema } = require('express-validator/check');
+
+const renderToRespone = require('./renderer');
+
+
+const PORT = process.env.NODE_PORT || 2929;
+
+const app = express();
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(expressValidator());
+
+const server = app.listen(PORT, () => {
+    console.log(`Graph server running on port ${PORT}`);
+});
+
+
+app.get('/monitoring-location/:siteID/', checkSchema({
+    renderer: {
+    },
+    parameterCode: {
+        in: ['query'],
+        errorMessage: 'parameterCode (5 digit integer) is required',
+        isInt: true,
```

Comment:
Yes - this will verify that the string corresponds to an integer, but not coerce it to an integer object (the `toInt` sanitizer would be used for that).